packages/react/src/components/UIShell/HeaderMenuItem.js

```diff
@@ -66,7 +70,13 @@ HeaderMenuItem.propTypes = {
   className: PropTypes.string,
 
   /**
-   * Applies selected styles to the item if a user sets this to true and aria-current !== 'page'.
+   * Applies selected styles to the item if a user sets this to true and `aria-current !== 'page'`.
+   */
+  isActive: PropTypes.bool,
+
+  /**
+   * Applies selected styles to the item if a user sets this to true and `aria-current !== 'page'`.
+   * @deprecated Please use `isActive`. This will be removed soon in v12.
    */
   isCurrentPage: PropTypes.bool,
```

Comment:
This needs the `deprecate()` prop types function helper applied here.

Here's an example of how it's used: https://github.com/carbon-design-system/carbon/blob/b0f39f3d34152057c1bd3f972ecd24e0c3269982/packages/react/src/components/ComboBox/ComboBox.tsx#L736-L740